crates/api/api_utils/src/utils.rs

```diff
@@ -320,15 +320,19 @@ pub fn check_comment_deleted_or_removed(comment: &Comment) -> LemmyResult<()> {
   }
 }
 
+/// Checks to see if a potential blocker can receive my message.
```

Comment:
```suggestion
/// Checks to see if a user can receive comment reply notification
```

Reply:
Also you can probably pass `&Post` instead, which already contains `community_id` and `community_instance_id`

Reply:
The creator id might need to be the comment creator id, not the post creator id.